docs/plugins/ingest-useragent.asciidoc

```diff
@@ -0,0 +1,66 @@
+[[ingest-useragent]]
+=== Ingest Useragent Processor Plugin
+
+The Useragent processor extracts details from the useragent string a browser sends with its web requests. 
```

Comment:
Sorry, I disagree with the reasoning used here. By favoring consistency with Logstash, the decision is inconsistent with standard Java naming conventions, internal Elasticsearch naming conventions, the rest of the Elasticsearch API, and has created an internally inconsistent plugin since the code uses UserAgent clashing with useragent elsewhere. If consistency is the reason, it's consistency with the Elasticsearch repository that should be favored. This decision should be revisited.